app/templates/index.hbs

```diff
@@ -0,0 +1,10 @@
+ <main>
+    <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
+    <h2 class="info-note highlighted-text">You can check out all our current events happening <a href="./events.html">here</a></h2>
+    <img class="logo-img" src="assets/images/Real-Dev-Squad@1x.png" alt="Real Dev Squad logo" />
+    <h3 class="info-note"> To know how this page is getting developed, join our Discord Channel. Contact one of the existing members </h3>
+  </main>
+  
+  <footer>
+    <p class="info-repo">The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website" target="_blank" rel="noopener noreferrer">open sourced repo</a></p>
+  </footer>
```

Comment:
Can you check if there is way we can leverage the Footer component here; since the text is the same

Reply:
Done!. Please review my PR again.